metricbeat/module/windows/_meta/config.yml

```diff
@@ -0,0 +1,13 @@
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 1s
+  counters:
+    - group: "processor"  
+      collectors:
+        - alias: "processor_perfomance"
+          query: "\\Processor Information(_Total)\\% Processor Performance"
```

Comment:
If you use single quotes here, can the double backslashes be replaced with a single backslash? I would prefer that, unless of course using double backslashes is common in the perfmon world.